sokoban.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'board.rb'
+require_relative 'person.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.